src/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/CmsSigner.cs

```diff
@@ -308,25 +302,16 @@ internal static List<AttributeAsn> BuildAttributes(CryptographicAttributeObjectC
 
             foreach (CryptographicAttributeObject attributeObject in attributes)
             {
-                using (var writer = new AsnWriter(AsnEncodingRules.DER))
+                AttributeAsn newAttr = new AttributeAsn
                 {
```

Comment:
AFAICT this is the only place where DER sorting rules were applied to the AttrValues. All callers of this function explicitly call `PkcsHelpers.NormalizeAttributeSet` after building the whole attribute set, so the sorting is still done in the end.